### PR TITLE
SAK-41197 Allow log job to include a stacktrace.

### PR DESCRIPTION
**Pull request by buckett**
The test logging job doesn't include a stacktrace, this adds the ability
to include a stacktrace in the test logging event.

### CODE REVIEW

File: jobscheduler/scheduler-component/src/webapp/WEB-INF/components.xml [hunk 1]
```diff
@@ -213,6 +213,12 @@
                    <property name="descriptionResourceKey" value="logger.description"/>
                    <property name="defaultValue" value="org.sakaiproject.component.app.scheduler.jobs.logmessage.LogMessageJob"/>
                </bean>
+               <bean class="org.sakaiproject.component.app.scheduler.jobs.SpringConfigurableJobProperty">
+                   <property name="required" value="true"/>
```
Comment (jonespm) [inline jobscheduler/scheduler-component/src/webapp/WEB-INF/components.xml]: This property doesn't "seem" like it's required?

Comment (buckett) [reply jobscheduler/scheduler-component/src/webapp/WEB-INF/components.xml]: Nice catch Matt, fixed.

**Comment by ern** [comment]
@buckett can you fix the conflict

**Comment by buckett** [comment]
Conflict resolved.

**Pull request merged**